### ISSUE TITLE
Stashing open tabs to new group without closing tabs does not work

### ISSUE THREAD

**Issue by frikdt** [josh-berry/tab-stash#438]
### System Details

- **OS Version:** Debian 11.7
- **Browser Version:** Firefox 102.10.0esr (64-bit)
- **Tab Stash Version:** 3.0

**Installed Browser Extensions:**

- None of the listed incompatible extensions is installed.

### Describe the Bug

When trying to stash open tabs without closing them, tab-stash does nothing.

### Steps to Reproduce

1. Open tab-stash in new tab or sidebar.
2. Hover over stash button; the following tool-tip is displayed: `Stash all open tabs to a new group (Hold Alt to keep tabs open)`
3. Hold down alt key (left or right) and click the `stash` button.
4. Nothing happens.

### Expected Behavior

I would expect a new tab group to appear, containing the list of open tabs.

### Impact

- I have tried to complete the operation using a separate tab-stash tab, as well as trying it from tab-stash opened in the side bar.
- I have tried with both Alt keys.

### Additional Details

<hr>
<em>Readers: If you are also experiencing this bug, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the issue if you have more details to add, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
Very strange! I'm not able to reproduce this on my system at all, so it would be helpful if you could see if there are any error logs produced when you click the button.

Did Tab Stash show you a message saying that a crash had occurred? (I assume not since you didn't mention it, but want to double-check.)

Since this happens in the Tab Stash tab, can you try collecting logs from there and paste them in here as a comment? Here's how: https://github.com/josh-berry/tab-stash/wiki/Collect-Error-Logs#to-collect-logs-from-the-tab-stash-tab-ui

**Comment by frikdt** [comment]
Hi Josh

Thanks for looking into this.

Tab Stash never showed any error messages. I tried collecting logs but the debugger console did not register anything.

However, since I had the debugger open, I added some breakpoints to the relevant event listener and then discovered that it is not being triggered at all (please see attached screenshots - and you may ignore the two warnings in the output - that happened while I unintentionally resized the window).

Here is the interesting part, however: I then decided to try a few other key combinations just for the heck of it - and as soon as I used `<Shift + Alt>` it worked. The event handler fires and the tabs are saved while remaining open.

So I don't know if this could be something peculiar to my system - although nothing obvious comes to mind; but I'm happy for this ticket to be closed.


Screenshots:

![tabstash--20240109T093941](https://github.com/josh-berry/tab-stash/assets/4082906/50209e78-2cba-4d65-899e-54d769e71277)
![tabstash--20240109T094119](https://github.com/josh-berry/tab-stash/assets/4082906/80012628-eb8a-4c59-b530-285f97de4ba1)

**Comment by josh-berry** [comment]
Really weird; I wonder if perhaps there's some other privileged extension installed that's messing with event handlers. I've seen this before—for example see issue #51. Dunno if this sounds familiar at all?

I'm happy to leave it open for a little bit—if you DO manage to find out what's going on, I'd love to hear!

**Comment by frikdt** [comment]
All of this has made me very curious, so I cloned the repo this morning and started playing around with the vue event modifiers and system modifier keys, and running the locally modified add-on, but nothing I tried made any difference.

However, while reading up more on the vue library, I found this link that eventually gave it away: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/altKey

In particular, the following:
>  Be aware that the browser can't always detect the alt key on some operating systems. On some Linux variants, for example, a left mouse click combined with the alt key is used to move or resize windows.

I then tried out `<Alt + MouseDown>` on a vim window, and sure enough it is used for moving the window; I wasn't even aware of this as I always move windows by grabbing the title bar (which does not require `Alt` to be pressed) or by using the cursor keys since I have *Window Move* bound to a keyboard shortcut in Xfce.

I then scrutinized the Xfce settings manager, and found this:

![image](https://github.com/josh-berry/tab-stash/assets/4082906/5a53507a-0f31-4b99-a706-0a30aaa323b9)

As soon as I changed the above window manager setting from *Alt* to *None* tab-stash started working normally.

With the window manager setting on *Alt* the combination `<Shift + Alt>` works, as I have mentioned previously. Just speculating, but I suppose it's possible that, in this case, the window manager passes `<Shift + Alt>` through to applications simply as `Alt`.

Either way, I think that this at least provides an explanation for the odd behavior.
